core/src/test/java/org/apache/iceberg/TestMetadataUpdateParser.java

```diff
@@ -360,6 +360,8 @@ public void testAddSnapshotToJson() throws IOException {
     long parentId = 1;
     long snapshotId = 2;
     int schemaId = 3;
+    Long firstRowId = 4L;
+    Long addedRows = 10L;
```

Comment:
can you add a separate test to this test class for the new metadata update that is being introduced?

Reply:
Sure I didn't have parsers for that yet, I just added in the change so that enabling row lineage would trigger a commit. I'll write that code now